README.md

```diff
@@ -1 +1,42 @@
-# Unity-Core
\ No newline at end of file
+# Unity-Core
```

Comment:
Can we name it `Ready Player Me Unity SDK Core` here? I was thinking in general if we should stuck SDK in the names of the packages and repo names too but I think later modules won't make sense to call SDK maybe. However, we need to make sure people understand at the first glance that this is a Unity-related project. What are your thoughts?

Reply:
Updated